### ISSUE TITLE
Update to  PostCSS 5.0

### ISSUE THREAD

**Issue by ai** [hallvors/postcss-flexboxfixer#4]

**Comment by hallvors** [comment]
@ai is this just updating the version number(s)?

**Comment by ai** [comment]
Nope. 5.0 has API changes https://github.com/postcss/postcss/releases/tag/5.0.0

**Comment by hallvors** [comment]
https://github.com/hallvors/postcss-flexboxfixer/commit/f1f2f1d1678ce042bee5be14d4acc2620b06b1bf
